app/bullets.lua

```diff
@@ -1,13 +1,14 @@
 Bullets = Object:extend()
 
-function Bullets:new()
+function Bullets:new(enemy)
   self.table = {}
+  self.enemy = enemy
```

Comment:
It seems unfortunate that Bullets now has to have a hard-link to enemy.  It would be nice to refactor to something more like the reactor pattern later.